examples/js/loaders/GCodeLoader.js

```diff
@@ -0,0 +1,211 @@
+'use strict';
+
+/**
+ * THREE.GCodeLoader is used to load gcode files usually used for 3D printing or CNC applications.
+ *
+ * Gcode files are composed by commands used by machines to create objects.
+ *
+ * @class THREE.GCodeLoader
+ * @param {Manager} manager Loading manager.
+ * @author tentone
+ * @author joewalnes
+ */
+THREE.GCodeLoader = function ( manager ) {
+
+	this.manager = ( manager !== undefined ) ? manager : THREE.DefaultLoadingManager;
+
+}
```

Comment:
Sorry for being so pedantic 😇 . Missing semicolon here...

Reply:
No problem i think you should be :+1:,